src/AccordionItemTitle/index.js

```diff
@@ -27,6 +28,7 @@ export default function AccordionItemTitle({
   return (
     <Root
       aria-controls={`react-sanfona-item-body-${uuid}`}
+      aria-expanded={expanded}
```

Comment:
I think the 'aria-expanded' should be set on this component; see W3C guidelines:

"When authors use collapsible content, for example, to hide navigation menus or lists of content, the triggering link or button should indicate to screen reader users whether the collapsable content below is in the expanded or in the collapsed state. The aria-expanded attribute is used for this purpose."

https://www.w3.org/WAI/GL/wiki/Using_aria-expanded_to_indicate_the_state_of_a_collapsible_element